configure/executors/k8s/executor.Deployment.yaml

```diff
@@ -30,7 +30,7 @@ spec:
       serviceAccountName: executor
       containers:
         - name: executor
-          image: index.docker.io/sourcegraph/executor-kubernetes:insiders@sha256:0644f1e165fe3dac6cfea1f36a14596fd52d88200385d506d2746b6d522029c3
+          image: index.docker.io/sourcegraph/executor-kubernetes:insiders@sha256:83255544f6837576419008008f7a4b7c19cad1831001ba7176311930539d3d9e
```

Comment:
for follow-up: @eseliger is it alright that we don't deploy a tagged image here?